On Alpine, there is no localtime in available_timezones() causing a key
lookup error. Note: This script is only used for generating the
`src/icalendar/timezone/equivalent_timezone_ids_result.py` in prepare().

--- icalendar-6.1.1.orig/src/icalendar/timezone/equivalent_timezone_ids.py
+++ icalendar-6.1.1/src/icalendar/timezone/equivalent_timezone_ids.py
@@ -62,7 +62,6 @@
     """
     print(create_timezones, name)
     unsorted_tzids = available_timezones()
-    unsorted_tzids.remove("localtime")
     unsorted_tzids.remove("Factory")
     
     class TZ(NamedTuple):
